src/datastore/postgres-store.ts

```diff
@@ -2228,13 +2232,23 @@ export class PgDataStore
     });
   }
 
-  getBlock(blockIdentifer: { hash: string } | { height: number }): Promise<FoundOrNot<DbBlock>> {
+  getBlock(
+    blockIdentifer:
+      | { hash: string }
+      | { height: number }
+      | { burnBlockHash: string }
+      | { burnBlockHeight: number }
+  ): Promise<FoundOrNot<DbBlock>> {
     return this.query(client => this.getBlockInternal(client, blockIdentifer));
   }
 
   async getBlockInternal(
     client: ClientBase,
-    blockIdentifer: { hash: string } | { height: number }
+    blockIdentifer:
+      | { hash: string }
+      | { height: number }
+      | { burnBlockHash: string }
+      | { burnBlockHeight: number }
```

Comment:
Looks like we've reached critical mass on this type, it should be defined, probably in `src/datastore/common.ts`, e.g.
```ts
export type BlockIdentifier =
      | { hash: string }
      | { height: number }
      | { burnBlockHash: string }
      | { burnBlockHeight: number };
```